tests/ad/ad_ldap_directory.json

```diff
@@ -0,0 +1,912 @@
+{
+    "entries": [
```

Comment:
Fortunately, yes :)

Here's an example from the [docs](http://ldap3.readthedocs.io/mocking.html):
```
from ldap3 import Server, Connection, ALL_ATTRIBUTES
server = Server('my_real_server')
connection = Connection(server, 'cn=my_real_user,ou=test,o=lab', 'my_real_password', auto_bind=True)
if connection.search('ou=test,o=lab', '(objectclass=*)', attributes=ALL_ATTRIBUTES):
    connection.response_to_file('my_entries.json', raw=True)
```